Parse-Dashboard/browser-control/README.md

```
@@ -0,0 +1,729 @@
+{
+  "success": true
+}
+```
+
+## WebSocket Event Stream
+
+Connect to the WebSocket stream to receive real-time events from the browser.
+
+### Connection
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)